releases/nightly-build/compose-files/docker-compose.yml

```diff
@@ -0,0 +1 @@
+docker-compose-nexus-mongo.yml
```

Comment:
I like this composite way to select which version of compose file to use! 👍

Reply:
This probably shouldn't have been included in the PR.

Reply:
I vote for this way :-)

Reply:
Like it, but yes it should be in a separate PR.

Reply:
got cha

Reply:
whoops sorry. I did a lazy `git add .` and missed that it grabbed this :)

Reply:
We like this so much we thought it would be a great idea if you added docker-compose.yml to the .gitignore.